app/services/AdjacentCellsCalculator.coffee

```diff
@@ -11,15 +12,16 @@ class AdjacentCellsCalculator
       for j in [@x - 1, @x, @x + 1]
         continue if i is @y and j is @x
```

Comment:
@bmwwoo This line was changed to `unless i is @y and j is @x` in a previous PR. You might need to merge into master.